apps/nuxt3-ssr/components/VariableCard.vue

```diff
@@ -14,6 +14,10 @@ const props = withDefaults(
 const variableKey = computed(() => getKey(props.variable));
 
 const resourcePathId = resourceIdPath(variableKey.value);
+
+function repeats(variable: IVariable) {
```

Comment:
the number of repeats as before. E.g. when the variable defines are measurements from week 5 - 10 THEN repeats = 5 <= I now see my mistake. Is actually 4?

You might see in popup there is shown as 'week 5 - 10'. 
We could replace with this sentence, is just a bit longer but much more useful. What do you think? Otherwise we can consult Brenda or Marije

Reply:
I do agree with you actually. I was just reinstating previous behavior. I will give at a go to make it more meaningfull